Core/src/OffConverter/OffReader.cpp

```diff
@@ -88,8 +88,47 @@ std::shared_ptr<OffModel> OffReader::readFile(const std::string& filename)
 		}
 		model->addVertices(allVertices);
 
-		//read faces
-		//... still to be added
+		//read faces (special case: after indices of face the color is given in RGB value -> to be considered later on)
```

Comment:
please denote this comment as an issue when this pull request is merged - for further consideration.

Reply:
I will do so